META.coq.in

```diff
@@ -68,7 +68,7 @@ package "kernel" (
 
   directory   = "kernel"
 
-  requires    = "dynlink, coq.lib, coq.vm"
+  requires    = "coq.lib, coq.vm"
```

Comment:
In the META file generated by dune, `coq.clib` and `coq.config` are also listed here.

Reply:
Good catch, I've adjusted the Dune deps here.